examples/data-transfer-mssql/README.md

```diff
@@ -0,0 +1,53 @@
+# MSSQL Example
+
+Example project using mssql with pyodbc docker image. which take a data from
+one db and insert to another table
```

Comment:
Example project to demonstrate a data pipeline container built using `laudio/pyodbc` as a base image - that transfers data from one database to another database.